tsconfig.json

```
@@ -65,6 +65,13 @@
     /* Advanced Options */
     "skipLibCheck": true,                     /* Skip type checking of declaration files. */
     "forceConsistentCasingInFileNames": true,  /* Disallow inconsistently-cased references to the same file. */
-    "resolveJsonModule": true
+    "resolveJsonModule": true,
+    "lib": [
+      "dom",
+      "es5",
+      "scripthost",
+      "es2015.collection",
+      "es2015.promise"
+    ]
```

Not sure what this is for but should probably not be committed
```suggestion
    "resolveJsonModule": true
```

---

Not sure what this is either, something WebStorm auto-generated I think. Should I add it to gitignore?